bundles/bundle_uk_UA.properties

```diff
@@ -0,0 +1,464 @@
+Blocks
+## Campaign
+
+## Crafter
+block.exotic-mod-advanced-melter.name = Покращена плавильня
+block.exotic-mod-advanced-melter.description = Виробляє дуже нестабільну рідину. Утримуйте її з обережністю. Один неправильний рух і вона вивергнеться. Не дайте їй витікати.
+block.exotic-mod-amethyst-refiner.name = Очищувач аметистів
+block.exotic-mod-amethyst-refiner.description = Вирізує аметисти в кристальні пластини
+block.exotic-mod-extractor.description = Витягує анден який залишився в брухті. Потребує води щоб зламати брухт.
+block.exotic-mod-furnace.name = Піч
+block.exotic-mod-furnace.description = Сплавляє кремній та свинець разом щоб отримати территовий сплав.
+block.exotic-mod-industrial-compressor.name = Індустріальний компрессор
+block.exotic-mod-industrial-compressor.description = Стискає пластаній набагато швидше, ніж звичайний компрессор. Потребує стручки спор для нафти.
+block.exotic-mod-purifier.name = Очищувач баєніту
+block.exotic-mod-purifier.description = Фільтрує та очищує баєніт.Частина води втрачається через випаровування.
+block.exotic-mod-refinery.name = Очищувач
+block.exotic-mod-refinery.description = Масивна установка виплавки сплавів. Виробляє намагнічений шлак як побічний продукт і вимагає занурення у воду для охолодження.
+block.exotic-mod-shadow-infuser.name = Тіньовий сплавлювач
+block.exotic-mod-shadow-infuser.description = Забирає велику кількість сучасних матеріалів і поєднує їх властивості для створення надзвичайно потужного сплаву.
+block.exotic-mod-supercoolant-mixer.name = Перемішувач суперохлоджувальної рідини
+block.exotic-mod-supercoolant-mixer.description = Додає дрібний порошок аметисту з кріогенною рідиною для створення більш ефективної охолоджуючої рідини.
+block.exotic-mod-tri-weaver.name = Трьохфазовий ткач
+block.exotic-mod-tri-weaver.description = Покращена версія фазофого ткача. Виробляє великі об'єми фазової тканини з кращою еффекетивністю. Потребує охолоджувача.
+block.exotic-mod-tungsten-refiner.name = Очищувач вольфрама
+block.exotic-mod-tungsten-refiner.description = Витягує вольфрам з вольфрамітового брухту для використання в юнітах.
+block.exotic-mod-wolframite-converter.name = Вольфрамітовий конвертер
+block.exotic-mod-wolframite-converter.description = Конвертує вольфрамітовий брухт назад в синьовольфраміт
+
+## Defence
+block.exotic-mod-absorber-wall.name = Поглинаюча стіна
+block.exotic-mod-absorber-wall.description = Слабкий захисний блок, здатний відновлюватися за рахунок енергії.
+block.exotic-mod-absorber-wall-large.name = Велика поглинаюча стіна
+block.exotic-mod-absorber-wall-large.description = Слабкий захисний блок, здатний відновлюватися за рахунок енергії.
```

Comment:
https://onlinecorrector.com.ua/uk/%D0%B7%D0%B0-%D0%BA%D0%BE%D1%88%D1%82-%D0%B7%D0%B0%D0%B2%D0%B4%D1%8F%D0%BA%D0%B8-%D0%B2%D0%BD%D0%B0%D1%81%D0%BB%D1%96%D0%B4%D0%BE%D0%BA/